lib/livebook_web/live/session_live/live_widget/vega_lite_live.ex

```diff
@@ -0,0 +1,32 @@
+defmodule LivebookWeb.SessionLive.LiveWidget.VegaLiteLive do
+  use LivebookWeb, :live_view
```

Comment:
I made this as a LV instead of component after all, so that the widget server can detect once it goes down by monitoring it.